src/Thermal/RadwsEOS.cpp

```diff
@@ -70,7 +71,8 @@ FLOAT Radws<ndim>::Pressure(Particle<ndim> &part)
 template <int ndim>
 FLOAT Radws<ndim>::EntropicFunction(Particle<ndim> &part)
 {
-  return (part.gamma - 1.0)*part.u*pow(part.rho, (FLOAT) (1.0 - part.gamma));
+  FLOAT gamma = opacity_table->GetGamma(part);
```

Comment:
I'm not sure this is correct, but I don't think we use it, so I'm happy to through an exception.